flash-messages/views/chirp_new.erb

```diff
@@ -0,0 +1,24 @@
+<section id="chirps-new-page">
+  <h2>Write a Chirp</h2>
+
+  <% if errors = get_flash(:errors) %>
+    <div class="form-errors">
+      <h3>Uh oh! There were some errors.</h3>
+      <ul class="error-messages">
+        <% errors.each do |error_msg| %>
+          <li><%= error_msg %></li>
+        <% end %>
+      </ul
+    </div>
```

Comment:
Side note: I prefer we re-render the page and pass in a local variable for the model for rendering errors; as passing them through flash is a minor anti-pattern